<launch>
  <arg name="map_file" />
  <arg name="map_name" />

  <param name="warehouse_host" value="localhost"/>
  <param name="warehouse_port" value="27017"/>  <!-- 27017 is default, use a private one to avoid error 48 conflicts -->
  <node  pkg="warehouse_ros" type="mongo_wrapper_ros.py" name="mongo">
    <param name="database_path" value="warehouse_data"/>
    <param name="overwrite" value="false"/>
  </node>

  <node pkg="world_canvas_server" type="annotations_server" name="annotations_server" output="screen">
     <!-- unused topics -->
     <!-- <remap from="dynamic_map" to="annotations_store_dynamic_map"/>
     <remap from="map" to="annotations_store_map"/> -->
  </node> 
</launch>
